shared/constants/transaction.ts

```diff
@@ -163,6 +163,12 @@ export const IN_PROGRESS_TRANSACTION_STATUSES = [
   TransactionStatus.signed,
   TransactionStatus.submitted,
   TransactionStatus.pending,
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  TransactionStatus.rejected,
+  TransactionStatus.failed,
+  TransactionStatus.dropped,
+  TransactionStatus.confirmed,
```

Comment:
Why are these statuses considered "In progress"  these four specifically refer to transactions that are completed on network?

Reply:
They aren't really, it was just to not create a new group, but yeah this can lead to some confusion, so we'll just create a new group called FINALIZED_TRANSACTION_STATUSES and add them there.

Reply:
the statuses in this group are used for pending transactions in various places in the code that would have to be further customized to treat these statuses differently... so yeah using a new group is preferred.